CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## v0.7.2
+
+* Updates
+  * Connection status message callbacks 
```

Comment:
Might want to rewrite as "Support querying the session's connection status"

Reply:
That's significantly more clear. Thank you!